### PR TITLE
Add BootstrapToken info for supported systems

### PR DESCRIPTION
**Pull request by poundbangbash**
macOS 10.15 introduced BootstrapTokens for handing off SecureTokens to accounts on login.
This PR references the output of `profiles status -type bootstraptoken` to evaluate if:

1. Bootstrap Tokens are supported (assigned to an MDM in ABM/ASM), and
2. The Bootstrap Token is escrowed to the MDM

This adds:

- Listing Columns
- Client details in the existing FileVault detail
- Two widgets in the FileVault report

### CODE REVIEW

File: migrations/2020_10_21_000001_filevault_status_add_bootstrap_columns.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            $table->index('bootstraptoken_escrowed');
+        });
+        
+    }
+
+    public function down()
+    {
+        $capsule = new Capsule();
+        $capsule::schema()->table($this->tableName, function (Blueprint $table) {
+            $table->dropColumn('auth_restart_support');
```
Comment (tuxudo) [inline migrations/2020_10_21_000001_filevault_status_add_bootstrap_columns.php:32]: Going down a version shouldn't drop these columns for this migration. Only the columns the migration adds should be dropped

File: scripts/filevaultstatus [hunk 2]
```diff
@@ -214,6 +214,37 @@ def get_filevault_active():
...
+    output = output.splitlines()
+
+    for line in output:
+        if "supported on server: YES" in line:
+            return {'bootstraptoken_supported': 1}
+    #If supported is not YES, return false
+    return {'bootstraptoken_supported': 0}
+
+
+def get_bootstraptoken_escrowed():
```
Comment (tuxudo) [inline scripts/filevaultstatus:234]: These two functions should be combined to prevent running the command more than once

**Comment by poundbangbash** [comment]
Good catch on the migration - copy pasta fail.

I updated the client script to capture both in one run.
I'm not sure what happened on the last commit but the files look ok.

**Comment by tuxudo** [comment]
Looks good! I'm not a maintainer on this module, but I'd merge it if I was :D

**Pull request merged**